src/main/java/net/mcreator/ui/minecraft/potions/JPotionListEntry.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2021, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.minecraft.potions;
+
+import net.mcreator.element.parts.EffectEntry;
+import net.mcreator.element.types.PotionItem;
+import net.mcreator.minecraft.ElementUtil;
+import net.mcreator.ui.MCreator;
+import net.mcreator.ui.component.util.PanelUtils;
+import net.mcreator.ui.help.HelpUtils;
+import net.mcreator.ui.help.IHelpContext;
+import net.mcreator.ui.init.L10N;
+import net.mcreator.ui.init.UIRES;
+import net.mcreator.workspace.Workspace;
+
+import javax.swing.*;
+import java.awt.*;
+import java.util.List;
+
+public class JPotionListEntry extends JPanel {
+
+	private final JSpinner duration = new JSpinner(new SpinnerNumberModel(3600, 1, 72000, 1));
```

Comment:
You shouldn't put a maximal value for the duration.

Reply:
Well I did not, it's your code. If I don't put a maximal value, what do I set as maximal value?